website/docs/components/app-header/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,98 @@
+
+### Context switcher
+
+The context switcher allows users to switch between different global contexts within the product or application, such as navigating between different organizations or projects. 
+
+![Context switcher within the App Header](/assets/components/app-header/app-header-context-switcher.png)
+
+!!! Info
+
+This element only exists in Figma. For implementation in code, use the HDS [Dropdown](/components/dropdown) or, if necessary, create your own custom component styled after the Dropdown.
+!!!
+
+## Utility navigation
+
+### Help dropdown
+
+Use the help dropdown to provide users with access to support and helpful resources that can be easily accessible from anywhere within the application, e.g., links to the help center, documentation, or tutorials.
+
+Avoid placing non-help-related links within the help dropdown menu.
+
+![Help dropdown in the App Header](/assets/components/app-header/app-header-help-dropdown.png)
+
+### User dropdown
+
+The user dropdown gives users quick and easy access to their settings and preferences. The menu should contain links or actions related to the user's profile, settings, and/or preferences.
+
+Avoid placing links to unrelated pages within the user dropdown menu.
+
+![User dropdown in the App Header](/assets/components/app-header/app-header-user-dropdown.png)
+
+### Search
+
+Search within the App Header is triggered by a standard HDS [Button](/components/button) themed to match the App Header’s styles. The application teams are responsible for building search capabilities themselves.
+
+![Search in the App Header](/assets/components/app-header/app-header-search.png)
+
+## Sizes
+
+The App Header supports basic responsive behavior out of the box with a single breakpoint at 1088px. The Figma component offers two variants:
+
+- `Large`: applicable for most desktop-sized viewports with a minimum width of 1088px.
+- `Small`: applicable for most mobile and some tablet-sized viewports with a maximum width of 1088px.
+
+![Large size of the App Header](/assets/components/app-header/app-header-size-large.png)
+
+![Small size of the App Header](/assets/components/app-header/app-header-size-small.png)
+
+## Menu
+
+In the Ember component, the controls (context switcher, help/user dropdown, etc) contained within the App Header will collapse automatically into a vertical menu that can be toggled open and closed with the Menu button.
+
+In Figma, we publish a separate Menu component for UIs and prototypes for smaller viewports.
+
+![Example of the App Header menu at smaller viewports](/assets/components/app-header/app-header-menu.png)
+
+### Responsive characteristics
+
+By default (and if used within the App Frame), the App Header will occupy 100% of the viewport width. As the viewport width condenses, the components' controls will collapse into a menu that can be toggled open/closed with the menu button (shown conditionally based on the viewport width).
```

Comment:
Here we're referring to the Ember components, should we use the corresponding terms then? 

(this is a classic example why I think having different "Side Nav/App Header" and `SideNav/AppHeader` is not really beneficial, but that's a personal opinion)

Reply:
It's difficult to remember to add/not add a space within component names in different contexts.